packages/commerce-sdk-react/src/hooks/ShopperOrders/mutation.ts

```diff
@@ -81,16 +83,36 @@ export const SHOPPER_ORDERS_NOT_IMPLEMENTED = [
     'UpdatePaymentInstrumentForOrder'
 ]
 
+type UseShopperOrdersMutationHeaders = NonNullable<
+    Argument<Client[ShopperOrdersMutationType]>
+>['headers']
+type UseShopperOrdersMutationArg = {
+    headers?: UseShopperOrdersMutationHeaders
+    rawResponse?: boolean
+}
+
 /**
  * A hook for performing mutations with the Shopper Orders API.
  */
-export function useShopperOrdersMutation<Action extends ShopperOrdersMutationType>(action: Action) {
+function useShopperOrdersMutation<Action extends ShopperOrdersMutationType>(
+    action: Action,
+    arg?: Omit<UseShopperOrdersMutationArg, 'rawResponse'> & {rawResponse?: false}
+): UseMutationResult<DataType<Client[Action]>, Error>
+function useShopperOrdersMutation<Action extends ShopperOrdersMutationType>(
+    action: Action,
+    arg?: Omit<UseShopperOrdersMutationArg, 'rawResponse'> & {rawResponse: true}
+): UseMutationResult<Argument<Client[Action]>, Error>
+function useShopperOrdersMutation<Action extends ShopperOrdersMutationType>(
+    action: Action,
+    arg?: UseShopperOrdersMutationArg
+): UseMutationResult<DataType<Client[Action]>, Error, Argument<Client[Action]>> {
     if (SHOPPER_ORDERS_NOT_IMPLEMENTED.includes(action)) {
         NotImplemented()
     }
     type Params = Argument<Client[Action]>
     type Data = DataType<Client[Action]>
     const queryClient = useQueryClient()
+
     return useMutation<Data, Error, Params>(
         (params, apiClients) => {
```

Comment:
Following the previous comment: this callback will receive `params`, which are whatever that is passed into `createOrderMutation.mutate(...)`.